### ISSUE TITLE
Load function in REPL (and context menu)

### ISSUE THREAD

**Issue by hansgru** [cursiveclojure/cursive#801]
There's *Tools/Load File in REPL* that allows to sync the changes, and it's quite useful.

However, when experimenting, it would be even more useful if there were a **Load Function in REPL** that would load only the function definition at the cursor in the editor.

It would be also very useful if this functionality could be called from the context menu too, not just from the main menu.

Thank you.

**Comment by cursiveclojure** [comment]
That sounds like Tools->REPL->Run top form in REPL - is that what you're after?

**Comment by hansgru** [comment]
> That sounds like Tools->REPL->Run top form in REPL - is that what you're after?

Yes :), but this should be totally in the context menu and also should behave context dependent. Regarding it's naming, it would be more consistent if it were also "Load xxx" not "Run".
"top form" might be precise and generic, something more concrete would be even more helpful, e.g. 
"Load "(top-function ...)" in the REPL"

**Comment by cursiveclojure** [comment]
Yes, except that the top form isn't always a function - it might be a protocol, a namespace, or even something like a string. I'll look into adding it to the context menu too.

**Comment by hansgru** [comment]
> "Yes, except that the top form isn't always a function - it might be a protocol, a namespace, or even something like a string."

Of course. I was just trying to say that simply displaying the form's name is more helpful than just saying "top form" (more like the Run Command where the name of what's being executed is also displayed).

> I'll look into adding it to the context menu too.

Thank you. Can't wait to try it :).

**Comment by cursiveclojure** [comment]
Ah, in the context-sensitive case, you mean - I see. I'll see about adding that, too.

**Comment by cursiveclojure** [comment]
This is fixed in the next build. In order to not create more confusion by using "load" again (since it's already used for files), I changed the text to "Send top form to REPL". The name is also context sensitive using a truncated version of the form when applicable. All the REPL actions are available in the editor popup under a "REPL" submenu (as in the Tools menu).

I also fixed a bunch of other REPL actions to do the same, including "Run test under caret".
